comp-bot/src/main/java/frc/robot/lights/LightsSubsystem.java

```diff
@@ -46,47 +50,60 @@ protected LightsState getNextState(LightsState currentState) {
+
+        case ERROR -> candle.setControl(new StrobeAnimation(0, 0).withColor(new RGBWColor(Color.kRed)).withFrameRate(BLINK_FAST_FRAMERATE));
+        case UNHOMED -> candle.setControl(new StrobeAnimation(0, 0).withColor(new RGBWColor(Color.kWhite)).withFrameRate(BLINK_SLOW_FRAMERATE));
+        case HOMED_NO_TAGS -> candle.setControl(new SolidColor(0, 0).withColor(new RGBWColor(Color.kYellow)));
+        case HOMED_SEES_TAGS -> candle.setControl(new SolidColor(0, 0).withColor(new RGBWColor(Color.kGreen)));
+
+        case INTAKING_CORAL -> candle.setControl(new StrobeAnimation(0, 0).withColor(new RGBWColor(Color.kWhite)).withFrameRate(BLINK_SLOW_FRAMERATE));
+        case INTAKING_ALGAE -> candle.setControl(new StrobeAnimation(0, 0).withColor(new RGBWColor(Color.kTeal)).withFrameRate(BLINK_SLOW_FRAMERATE));
+
+        case IDLE_EMPTY -> candle.setControl(new SolidColor(0, 0).withColor(new RGBWColor(Color.kBlack)));
+        case HOLDING_CORAL -> candle.setControl(new SolidColor(0, 0).withColor(new RGBWColor(Color.kWhite)));
+        case HOLDING_ALGAE -> candle.setControl(new SolidColor(0, 0).withColor(new RGBWColor(Color.kTeal)));
+
+        case LOLLIPOP_SEES_ALGAE -> candle.setControl(new SolidColor(0, 0).withColor(new RGBWColor(Color.kTeal)));
+        case LOLLIPOP_NO_ALGAE -> candle.setControl(new SolidColor(0, 0).withColor(new RGBWColor(Color.kBlack)));
+
+        case CORAL_HANDOFF -> candle.setControl(new StrobeAnimation(0, 0).withColor(new RGBWColor(Color.kWhite)).withFrameRate(BLINK_FAST_FRAMERATE));
+
+        case CLIMB_LINEUP -> candle.setControl(new SolidColor(0, 0).withColor(new RGBWColor(Color.kYellow)));
+        case CLIMB_HANG -> candle.setControl(new SolidColor(0, 0).withColor(new RGBWColor(Color.kGreen)));
+        case CLIMB_STOP -> candle.setControl(new StrobeAnimation(0, 0).withColor(new RGBWColor(Color.kGreen)).withFrameRate(BLINK_SLOW_FRAMERATE));
+
+        case SCORE_NO_ALIGN_NO_TAGS -> candle.setControl(new StrobeAnimation(0, 0).withColor(new RGBWColor(Color.kYellow)).withFrameRate(BLINK_SLOW_FRAMERATE));
+        case SCORE_NO_ALIGN_TAGS -> candle.setControl(new StrobeAnimation(0, 0).withColor(new RGBWColor(Color.kGreen)).withFrameRate(BLINK_SLOW_FRAMERATE));
+        case SCORE_ALIGN_NO_TAGS -> candle.setControl(new StrobeAnimation(0, 0).withColor(new RGBWColor(Color.kYellow)).withFrameRate(BLINK_FAST_FRAMERATE));
+        case SCORE_ALIGN_TAGS -> candle.setControl(new StrobeAnimation(0, 0).withColor(new RGBWColor(Color.kGreen)).withFrameRate(BLINK_FAST_FRAMERATE));
+
+        case SCORING_ALGAE -> candle.setControl(new StrobeAnimation(0, 0).withColor(new RGBWColor(Color.kTeal)).withFrameRate(BLINK_FAST_FRAMERATE));
+        case SCORING_CORAL -> candle.setControl(new StrobeAnimation(0, 0).withColor(new RGBWColor(Color.kWhite)).withFrameRate(BLINK_FAST_FRAMERATE));
+
+        case OTHER -> candle.setControl(new StrobeAnimation(0, 0).withColor(new RGBWColor(Color.kPurple)).withFrameRate(BLINK_SLOW_FRAMERATE));
```

Comment:
instead of creating a new control request every time, just have one solid request and one strobe request, then reuse those. look at how other subsystems reuse their control requests (ex. a single motion magic request where the position is updated)